v-next/hardhat/test/internal/builtin-plugins/solidity/config.ts

```diff
@@ -570,7 +571,7 @@ describe("solidity plugin config resolution", () => {
       );
       assert.equal(
         resolvedConfig.solidity.profiles.production.preferWasm,
-        true,
+        true && shouldUseWasm(),
       );
```

Comment:
```suggestion
```

I think this assertion and the reference to `production` above should be deleted - or we should reword the name of the test.